architecture/mce-overview-ocp.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: ASSEMBLY
+[id="mce-overview-ocp"]
+= About multicluster engine for Kubernetes operator
+include::_attributes/common-attributes.adoc[]
+:context: mce-overview-ocp
+
+toc::[]
+
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator (MCE). The operator delivers full lifecycle capabilities for managed {product-title} clusters and partial lifecycle management for other Kubernetes distributions. It is available in two ways:
+
+* As a standalone operator that you install as part of your {product-title} or {oke} subscription
+* As part of link:https://access.redhat.com/products/red-hat-advanced-cluster-management-for-kubernetes[Red Hat Advanced Cluster Management for Kubernetes]
+
+[id="mce-on-ocp"]
+== Cluster management with multicluster engine on {product-title}
+
+When you enable multicluster engine on {product-title}, you gain the following capabilities:
+
+* xref:control-plane.adoc#hosted-control-planes-overview_control-plane[Hosted control planes], which is a feature that is based on the HyperShift project. With a centralized hosted control plane, you can operate {product-title} clusters in a hyperscale manner.
+* Hive, which provisions self-managed {product-title} clusters to the hub and completes the initial configurations for those clusters.
+* klusterlet agent, which registers managed clusters to the hub.
+* Infrastructure Operator, which manages the deployment of the Assisted Service to orchestrate on-premise bare metal and vSphere installations of {product-title}, such as SNO on bare metal. The Infrastructure Operator includes xref:../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.adoc#ztp-challenges-of-far-edge-deployments_ztp-deploying-far-edge-clusters-at-scale[zero touch provisioning (ZTP)], which fully automates cluster creation on bare metal and vSphere provisioning with GitOps workflows to manage deployments and configuration changes.
+* Open cluster management, which provides resources to manage Kubernetes clusters.
+
+The multicluster engine is included with your {product-title} support subscription and is delivered separately from the core payload. To start to use multicluster engine, you deploy the {product-title} cluster and then install the operator. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/multicluster_engine_overview#mce-install-intro[the installation documentation].
+
+[id="mce-on-rhacm"]
+== Cluster management with Red Hat Advanced Cluster Management
+
+If you need cluster management capabilities beyond what {product-title} with multicluster engine can provide, consider Red Hat Advanced Cluster Management. The multicluster engine is an integral part of Red Hat Advanced Cluster Management and is enabled by default. 
```

Comment:
should we link to the ACM landing page or doc here?

Reply:
I linked to the ACM landing page at the first mention of ACM near the beginning of the topic.